### PR TITLE
profile upgrade experience

### PR DESCRIPTION
**Pull request by michaelkirk**
PTAL @charlesmchen
 
profile-iphone5c

### CODE REVIEW

File: Signal/src/ViewControllers/ExperienceUpgradesPageViewController.swift [hunk 1]
```diff
@@ -4,6 +4,94 @@
...
+
+        // Title label
+        let titleLabel = UILabel()
+        view.addSubview(titleLabel)
+        titleLabel.text = header
+        titleLabel.textAlignment = .center
+        titleLabel.font = UIFont.ows_regularFont(withSize: ScaleFromIPhone5(24))
+        titleLabel.textColor = UIColor.white
+        titleLabel.minimumScaleFactor = 0.5
+        titleLabel.adjustsFontSizeToFitWidth = true
```
Comment (charlesmchen) [inline Signal/src/ViewControllers/ExperienceUpgradesPageViewController.swift]: Can you please attach screenshots on plus size device?

File: Signal/src/environment/ExperienceUpgradeFinder.swift [hunk 2]
```diff
@@ -6,27 +6,45 @@ import Foundation
...
-        }
-
-        return upgrades
+        return [
+            // Disable old experience upgrades. Most people have seen them by now, and accomodating multiple makes layout harder.
+            // Note if we ever want to show multiple experience upgrades again
+            // we'll have to update the layout in ExperienceUpgradesPageViewController
+            //
+            // videoCalling,
+            // (UIDevice.current.supportsCallKit ? callKit : nil),
```
Comment (charlesmchen) [inline Signal/src/environment/ExperienceUpgradeFinder.swift]: `ExperienceUpgradesPageViewController` still has paging... is that for future scenarios where we might want to have two pages?

Comment (michaelkirk) [reply Signal/src/environment/ExperienceUpgradeFinder.swift]: Correct.

File: Signal/src/ViewControllers/ExperienceUpgradesPageViewController.swift (hunk 1 above)
Comment (michaelkirk) [reply Signal/src/ViewControllers/ExperienceUpgradesPageViewController.swift]: screen shot 2017-09-05 at 4 28 28 pm

**Pull request merged**